letsencrypt/tests/auth_handler_test.py

```diff
@@ -148,6 +148,42 @@ def test_perform_failure(self):
         self.assertRaises(
             errors.AuthorizationError, self.handler.get_authorizations, ["0"])
 
+    @mock.patch("acme.challenges.HTTP01Response.simple_verify")
+    @mock.patch("acme.challenges.TLSSNI01Response.simple_verify")
+    def test_simple_verify(self, mock_http01_verify, mock_tlssni01_verify):
+        from acme import jose
+        from letsencrypt.tests import test_util
```

Comment:
I followed the pattern of a few other methods in the same file, wasn't sure which is more correct.